be/CMakeLists.txt

```diff
@@ -489,6 +489,61 @@ if ("${USE_STAROS}" STREQUAL "ON")
        )
 endif()
 
+# LLVM
+# TODO(yueyang): Adjust libraries during the implementation of JIT.
+set (LLVM_LIBRARIES
+    LLVMRuntimeDyld
+    LLVMBitstreamReader
+    LLVMOption
+    LLVMAsmPrinter
+    LLVMProfileData
+    LLVMAsmParser
+    LLVMOrcTargetProcess
+    LLVMExecutionEngine
+    LLVMX86CodeGen
```

Comment:
should this LLVMX86 libraries be moved to x86 conditions and not necessary for arm build?

Reply:
Yes this looks like a bug and I'll fix it.